src/modules/CRUD/components/Details/Read/Read.js

```diff
@@ -13,6 +13,28 @@ const NO_FEATURE = 'CRUD.details.noFeature';
 
 const emptyStringNullOrUndef = value => ['', null, undefined].includes(value);
 
+const isTableObject = value => (
+  value.reduce((list, item) => {
+    if (!list.columns.length) {
+      return {
+        ...list,
+        columns: Object.keys(item),
+        rows: [Object.values(item)],
+      };
+    }
+    if (!Object.keys(item).sort().every((prop, i) => list.columns.sort()[i] === prop)) {
```

Comment:
`if (whaaaaaats‽…) {`

Could you put this in a tiny pure function with a self explanatory name? _(which will probably take `item` & `list` as arguments)_


> « It's harder to read code than to write it »